PrebidMobile/PrebidMobile-rendering/src/main/java/org/prebid/mobile/rendering/sdk/PrebidRenderingSettings.java

```diff
@@ -91,7 +90,7 @@
      */
 
     public static void initializeSDK(Context context, final SdkInitListener sdkInitListener)
-    throws AdException {
+            throws AdException {
         Log.d(TAG, "Initializing OpenX SDK");
```

Comment:
In the scope of future work need to review appearance of the words "OpenX" and "Apollo" one more time and remove them.

Reply:
Already in progress in a separate branch.

Reply:
Already in progress in a separate branch.